src/profile/components/editingNotifications/useNotificationContent.ts

```diff
@@ -0,0 +1,65 @@
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Action } from '../../hooks/useProfileDataEditor';
+
+export type NotificationContent = { error: boolean; text: string };
+export type ContentHandlers = {
+  setSuccessMessage: (message: string, action?: Action) => void;
+  setErrorMessage: (message: string, action?: Action) => void;
+  clearMessage: () => void;
+  content: NotificationContent;
+};
+function useNotificationContent(): ContentHandlers {
+  const [content, setMessage] = useState<NotificationContent>({
+    error: false,
+    text: '',
+  });
+  const { t } = useTranslation();
+  const tryAgainKey = 'notification.tryAgain';
+  const getActionMessage = (action: Action, error: boolean): string => {
+    if (action === 'remove') {
+      return error
+        ? `${t('notification.removeError')} ${t(tryAgainKey)}`
+        : t('notification.removeSuccess');
+    }
+    if (action === 'save') {
+      return error
+        ? `${t('notification.saveError')} ${t(tryAgainKey)}`
+        : t('notification.saveSuccess');
+    }
+    return error
+      ? `${t('notification.genericError')} ${t(tryAgainKey)}`
+      : t('notification.genericSuccess');
+  };
+  const setSuccessMessage: ContentHandlers['setSuccessMessage'] = (
+    message,
```

Comment:
Is this message used? In BasicData it feels like it is always set to an empty string. For example in line 41, it seems the callers always use action, so the result is never the message directly? I wonder if it would simplify the code if removed that unused parameter?

Reply:
Good point, it was added for one situation which does not exist anymore. Will be removed

Reply:
Done.